stellarobservatory/utils/scc.py

```diff
@@ -0,0 +1,27 @@
+"""Utilities for strongly connected components"""
+import numpy
+from scipy.sparse import csr_matrix
+from scipy.sparse.csgraph import connected_components
+
+def get_strongly_connected_components(graph):
+    """
+    Get strongly connected components for a directed graph
+
+    The returned list of components is ordered by dependency, i.e.,
+    such that the nodes in the first component have no dependencies on
+    other components.
```

Comment:
Todo check if true ;-) And maybe say that this is usually referred to as the topological order?

Reply:
See https://en.wikipedia.org/wiki/Tarjan%27s_strongly_connected_components_algorithm#Additional_remarks. I added "topological order".